oxidation/libparsec/crates/types/src/id.rs

```diff
@@ -314,14 +335,31 @@ impl_debug_from_display!(HumanHandle);
 
 impl std::fmt::Display for HumanHandle {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        write!(f, "{} <{}>", self.label, self.email)
+        self.display.fmt(f)
+    }
+}
+
+impl AsRef<str> for HumanHandle {
+    fn as_ref(&self) -> &str {
+        &self.display
+    }
+}
+
+impl FromStr for HumanHandle {
```

Comment:
Better remove this:
- this is not used in the code
- the start/stop computation is fragile (it should be thoughtfully tested)
- we should never need to turn `foo <foo@example.com>` back to `HumanHandle`

edit: this is needed due to the bindings currently the HumanHandle object is converted back and forth into a string to be passed to javascript :(
(I'll have a look at improving this...)